### PR TITLE
Remove references to Madeline

### PR DESCRIPTION
**Pull request by mastfish**
I feel like the example would be best using http://www.ruby-doc.org/core-2.2.0/Marshal.html not madeline.

No need to pull in extra dependencies when the standard library has got us covered, :).

Maybe using redis is a bad example and it should just write to a file?

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -41,22 +41,21 @@ A Bayesian classifier by Lucas Carlson. Bayesian Classifiers are accurate, fast,
...
+classifier.train_interesting "here are some good words. I hope you love them"
+classifier.train_uninteresting "here are some bad words, I hate you"
+classifier.classify "I hate bad words and you" # returns 'Uninteresting'
+
+classifier_snapshot = Marshal.dump classifier
+# This is a bytestring, you can persist it anywhere you like
+Redis.current.save "classifier", classifier_snapshot
 
-Using Madeleine, your application can persist the learned data over time.
+# Later
```
Comment (parkr) [inline README.markdown]: Maybe mention persistence?

**Comment by parkr** [comment]
Yeah, Madeline is old and outdated. Maybe show both a file and Redis here?

**Comment by Ch4s3** [comment]
I agree with @parkr, and example would be a helpful change.

**Comment by parkr** [comment]
?

**Comment by mastfish** [comment]
Woops, pressed close when I meant to press comment.

I've updated this with a hopefully clearer explanation.

**Comment by parkr** [comment]
LGTM!

**Comment by mastfish** [comment]
@Ch4s3 , are any more changes to this PR needed?

**Comment by Ch4s3** [comment]
Looks good.

**Comment by parkr** [comment]
@Ch4s3, can you please add an item to the history file for this and reference this pr in your commit for that line? Thanks :)

**Pull request merged**